packages/bierzo-wallet/src/store/reducers/extension/types.ts

```diff
@@ -0,0 +1,7 @@
+import { Action } from 'redux';
```

Comment:
For now, those types can live perfectly fine in `reducers/extension/reducer.ts`, the main reason is reducer is the root of everything, and we have this type because we are using redux, and as you can imagine, having 22 lines on `reducer.ts` it is good to move there the type.

Or what do you think? What was the reason to slip those files?

Reply:
I don't mind